frontend/src/hooks/useTypingEffect.ts

```diff
@@ -1,79 +0,0 @@
-import { useEffect, useState } from "react";
```

Comment:
Yes, my alternative thought to the action banner was the `...`. The banner looked better in my head because I was also imagining a similar banner could be used to indicate what tools the agent is using or process (such as install status)  indicators.

I'm not *fully* convinced of my current implementation either, but it does make up for it in fast responses, which is why I was searching for an alternative to the typing effect in the first place